.github/workflows/npm-publish.yml

```diff
@@ -1,11 +1,12 @@
-name: Publish NPM Packages
+name: Publish NPM Packages & Deploy Docs
 
 on:
+
   release:
     types: [ created ]
 
 jobs:
-  test:
+  build:
```

Comment:
sorry my last comment was totally wrong 😄  test is the good job name.... its the workflow file `node.js.yml` wich is badly named.....

Reply:
i'll let hit like this it makes more sense